script/analysis/snap.py

```diff
@@ -42,6 +42,9 @@
 parser.add_argument('--linecolor',
                     type=str, default='k',
                     help='Linecolor of A-contours. Needed only if using --A-contours')
+parser.add_argument('--zorder',
+                    type=str, default='k',
```

Comment:
Should be `type=int` and `default=2`

Reply:
whoops! results of copy pasting the previous line to avoid typing, and then forgetting to update it completely. :)